src/app.ts

```diff
@@ -4,13 +4,29 @@ import serverErrorHandler from './error-handler';
 import AccesstokenServiceManager from './modules/access-token/access-token-manager';
 import AccountServiceManager from './modules/account/account-service-manager';
 import CommunicationServiceManager from './modules/communication/communication-service-manager';
+import TaskServiceManager from './modules/task/task-service-manager';
 import ConfigManager from './modules/config/config-manager';
 import ConfigService from './modules/config/config-service';
 import LoggerManager from './modules/logger/logger-manager';
 
 export default class App {
   private static app: Application;
 
+  private static async createRESTApiServer(): Promise<Application> {
```

Comment:
This change does belong to this PR.

Please rebase to ensure you are only checking in changes that are related to this PR